src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -36,18 +37,17 @@ def ivalue_to_constant(ivalue):
         assert ov_type.is_static(), "Can't deduce type for list"
         return op.Constant(ov_type, Shape([len(ivalue)]), ivalue).outputs()
 
-    if isinstance(ivalue, torch.Tensor) and ivalue.type() in pt_to_ov_type_map:
-        try:
-            ovshape = PartialShape(ivalue.size())
-            ovtype = pt_to_ov_type_map[ivalue.type()]
-            ov_const = op.Constant(ovtype, ovshape.get_shape(), ivalue.data_ptr())
-        except Exception:
-            # old variant that makes a slow data copying
-            warnings.warn("[ WARNING ] Constant wasn't able to convert from data_ptr.")
-            nvalues = ivalue.numpy()
-            ovtype = np_to_ov_type_map[str(nvalues.dtype)]
-            ovshape = PartialShape(nvalues.shape)
-            ov_const = op.Constant(ovtype, ovshape.get_shape(), nvalues.flatten().tolist())
+    if isinstance(ivalue, torch.Tensor):
+        if ivalue.ndim == 0:
+            assert str(ivalue.dtype()) in pt_to_ov_type_map, "Type is not known"
```

Comment:
```
>>> x = torch.Tensor(1)
>>> x.dtype
torch.float32
>>> x = torch.tensor(1)
>>> x.dtype
torch.int64
>>> x = torch.tensor(1.0)
>>> x.dtype
torch.float32
>>> x = torch.tensor(np.int32(1))
>>> x.dtype
torch.int32
```
Can this be better to refactor `pt_to_ov_type_map` to work with just `dtype` and fallback to "convert int32" in specific cases?